src/services/fs-service.ts

```diff
@@ -51,14 +55,19 @@ export class FSService {
     return path.resolve(this.getBundleDirectory(), ...pathSegments)
   }
 
-  public createFileFromTemplate(
-    pathSegments: string[],
-    templateFileName: string
-  ): void {
+  public createFileFromTemplate(pathSegments: string[], templateFileName: string, replaceDict?: ContentReplaceDictionary): void {
     const filePath = this.getBundleFilePath(...pathSegments)
-    const templateFileContent = fs.readFileSync(
-      path.resolve(__dirname, '..', '..', RESOURCES_FOLDER, templateFileName)
+    let templateFileContent = fs.readFileSync(
+      path.resolve(__dirname, '..', '..', RESOURCES_FOLDER, templateFileName),
+      replaceDict ? 'utf8' : null,
     )
+    if (replaceDict) {
+      const placeholders: string[] = Object.keys(replaceDict)
+      templateFileContent = placeholders.reduce((currContent: string, placeholder: string) => (
+        currContent.replace(new RegExp(placeholder, 'g'), replaceDict[placeholder])
+      ), templateFileContent as string)
```

Comment:
I think another approach is:
```
for (placeholder of placeholders) {
  templateFileContent = templateFileContent.replace(new RegExp(placeholder, 'g'), replaceDict[placeholder])
}
```
So not sure if we need to remove the `no-array-reduce` rule.

Reply:
better, I think `reduce` is not needed here

Reply:
done